dds_web/api/user.py

```diff
@@ -83,8 +83,12 @@ def post(self):
         try:
             existing_user = user_schemas.UserSchema().load({"email": email})
             unanswered_invite = user_schemas.UnansweredInvite().load({"email": email})
-        except sqlalchemy.exc.OperationalError as err:
-            raise ddserr.DatabaseError(message=str(err), alt_message="Unexpected database error.")
+        except (sqlalchemy.exc.SQLAlchemyError, sqlalchemy.exc.OperationalError) as err:
+            raise ddserr.DatabaseError(
+                message=str(err),
+                alt_message="Something happened while checking for existig account / active invite.",
+                pass_message=False,
+            )
```

Comment:
Add a test for this too. Search for mentions of `mock_sqlalchemyerror` in other tests to see an example.